staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/BUILD

```diff
@@ -0,0 +1,60 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
We do not use bazel anymore.

Reply:
Thanks, it was some leftovers from the old PR, I removed the file.